### ISSUE TITLE
How to navigate to app when click on list item

### ISSUE THREAD

**Issue by naveen-iiit** [Nailik/AppWidgetListView#1]
Hi there,

How can we navigate to the app when I click on list item in the widget.

Any idea?

**Comment by Nailik** [comment]
I'm not sure what you want, do you want to open another app or do you want to navigate inside your own app?

**Comment by naveen-iiit** [comment]
Yup, Nailik I want to open my app with the data in list item but I thought I found a way.

Thanks for your reply.
